src/instruction/power_decode.C

```diff
@@ -61,7 +61,7 @@ test_results_t power_decode_Mutator::executeTest() {
   // Decoding in 64-bit mode uses 32-bit registers
   auto const arch = Dyninst::Arch_ppc32;
```

Comment:
The original code checks whether we are testing for ppc32 or ppc64 and picks the archictecture. 
In the suggested patch it seems like we are hard-fixed using ppc64? 
In that case, should we still use 32 bit registers when decoding here? Or should this be updated to be Arch_ppc64 as well?

Reply:
This test only ever used Arch_ppc32. The power_cft test did previously test both architectures. I forgot to add a note about that. I have no idea how or why, but Dyninst incorrectly decodes the bclr instruction properties in 32-bit mode. That's quite interesting because the power_decode test still worked because it got the *registers* correct for the control flow targets. :shrug: